packages/components/src/Accordion/AccordionItemRegion.tsx

```diff
@@ -1,4 +1,5 @@
-import { HTMLAttributes, useEffect, useRef, useState } from 'react';
+import { HTMLAttributes } from 'react';
+import { useElementSize } from 'usehooks-ts';
```

Comment:
This lib looks useful—good find :)

Reply:
I actually just removed because I found an alternative to the issue. But I do agree with the usefulness of the lib :)